samples/AspNetODataSample.Web/AspNetODataSample.Web.csproj

```diff
@@ -88,16 +88,14 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.OData.Core, Version=7.11.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\sln\packages\Microsoft.OData.Core.7.11.1\lib\net45\Microsoft.OData.Core.dll</HintPath>
-      <Private>True</Private>
```

Comment:
I hope that `<Private>True</Private>` getting dropped is not a problem...

Reply:
I updated the packages from "Manage Nuget packages" window in visual studio. I wasn't in control of the change.